### DISCUSSION TITLE
Forem Language support

### DISCUSSION THREAD

**Discussion by Abdullah-Schahin** [forem/forem#17670] [Q&A] [answered]
Hey folks, I'm trying to setup forem for my community and would like to know if forem support Arabic language?

Any info or advice is highly appreciated.

**Comment by JennieOcken** [comment, marked as answer]
Apologies, we do not have Forem support for the Arabic language.  There is nothing stopping the Forem posts to be in any language however we do not have translations in place for the built-in Forem features and do not have R-L support.
